net-dns/dnsdist/files/dnsdist.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=dnsdist
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+# Note: when editing the ExecStart command, keep --supervised and --disable-syslog
+ExecStart=/usr/bin/dnsdist --supervised --disable-syslog -u dnsdist -g dnsdist
+
+Type=notify
```

Comment:
However, if you set `Type=notify`, then I think systemd will wait for the notification and then assume it failed to start and kill it. The simplest solution would be to install the file conditional to `use systemd`.